.github/workflows/composite/audit/action.yml

```diff
@@ -9,6 +9,15 @@ runs:
   #    /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (openssl)
   #    /usr/local/opt/openssl/lib/libssl.1.0.0.dylib (openssl)
   #  /Users/runner/work/Homebrew-Tap/Homebrew-Tap/Formula/powershell.rb: * powershell has broken dynamic library links:
+
+  # We are skipping `version` because of the following error:
+  #  Broken dependencies:
+  #  /Users/runner/work/Homebrew-Tap/Homebrew-Tap/Formula/powershell-preview.rb: * Stable: version 7.2.0-rc.1 is redundant with version scanned from URL
+  #    Error: 1 problem in 1 formula detected
+  #    Error: Stable: version 7.2.0-rc.1 is redundant with version scanned from URL
+  #    Error: Process completed with exit code 1.
+  #
+  # We use version to quickly determine if an update is needed and we don't want to remove it
```

Comment:
I'm confused by this statement as it seems to contradict line 13?  What does `skipping version` mean here?

Reply:
the error explains, this is an audit, if brew can determine the version, they say not to declare the version.  But the results are ... non-deterministic with our URLs.  I have to add and remove the version often only after the formula breaks.

But assume they have fixed that, I use the version in the update script to quickly determine if the formula needs updated.  I don't want to have to parse it out of the URL.  I just don't like their rule.

Reply:
```suggestion
  # We use the version property to quickly determine if an update is needed and we don't want to remove it as the version rule which is being skipped suggests
```